.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: go
+sudo: false
 
 go_import_path: go.uber.org/fx
-
 go:
   - 1.8.x
   - tip
```

Comment:
Actually, gimme doesn't know what to do with 1.9.x, and I don't want to use the beta explicitly. We can come back to this after 1.9 is released.

Separately, I *really* wish gimme supported `stable`, `previous-stable`, and `next-stable`, since so many projects have the same support policies.